src/GitHook/Command/FileCommand/PreCommit/CodeStyleFixCommand.php

```diff
@@ -42,10 +62,28 @@ public function run(CommandContextInterface $context): CommandResultInterface
     {
         $commandResult = new CommandResult();
 
-        $processDefinition = ['vendor/bin/phpcbf', $context->getFile(), '--standard=config/ruleset.xml'];
+        $processDefinition = [
+            static::BINARY_STYLE_FIXER,
+            $context->getFile(),
+            $this->getStandardRulesetOption(),
+        ];
+
         $process = $this->buildProcess($processDefinition);
         $process->run();
 
         return $commandResult;
     }
+
+    /**
+     * @return string
+     */
+    protected function getStandardRulesetOption(): string
+    {
+        $filepath = static::RULESET_LEGACY_PATH;
+        if (file_exists(PROJECT_ROOT . DIRECTORY_SEPARATOR . static::RULESET_PATH)) {
+            $filepath = static::RULESET_PATH;
+        }
+
+        return sprintf('%s=%s', static::OPTION_STANDARD, $filepath);
```

Comment:
Please check the Jira ticket

If the root file exists, no --standard must be added here at all

Reply:
Also, if the config/ one doesnt exist it shouldnt error with that legacy path, but instead a meaningful message should clarify to add root file instead.